objects.py

```diff
@@ -99,7 +210,24 @@ def tojson(self):
             "report_level": self.report_level,
             "skip_report": self.skip_report,
             "ignore_kit_validation": self.ignore_kit_validation,
-            "props": self.properties.tojson()
+            "props": self.properties._tojson()
+        }
+
+    def _totateconfig(self):
```

Comment:
We should test this using [`unittest`](https://docs.python.org/3/library/unittest.html#basic-example).